components/searchable_user_list/searchable_user_list.tsx

```diff
@@ -2,46 +2,47 @@
 // See LICENSE.txt for license information.
 /* eslint-disable react/no-string-refs */
```

Comment:
Also thanks for removing the Jquery and ReactDOM usage from this file. We have tickets around to remove those as well, so I can see if they can be linked to this PR too.

Since you've removed the string refs from this file, could you also remove this comment?

Reply:
removed it and got an error that says "Using string literals in ref attributes is deprecated."

Reply:
I think we were mistaken about removing the string refs in this file, there are still a few in here.
This ticket isn't about removing string refs, so you don't have to do anything here if you don't want.